cinema/views.py

```diff
@@ -46,7 +46,7 @@ class ActorViewSet(
 ):
     queryset = Actor.objects.all()
     serializer_class = ActorSerializer
-    authentication_classes = (TokenAuthentication,)
+    authentication_classes = (JWTAuthentication,)
```

Comment:
u've just removed it from genericviewset but decided to leave it here (and other views), can u explain why ?

Reply:
yeah, i had day so i didnt check that in setting i already have base  auth class